### PR TITLE
add ukrainian localization.

### PR DESCRIPTION
**Pull request by 3a4oT**

### CODE REVIEW

**Review by tanner0101 — COMMENTED** [review]
Thanks. It would be great if we could get a third party to verify / spellcheck this. I'll post in discord asking.

File: src/i18n/index.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ import VueI18n from 'vue-i18n';
 // then import the file and add it to the 'translations' const as seen below
 
 import en from './en';
+import uk from './uk'
```
Comment (joscdk) [inline src/i18n/index.js:9]: Isn't the official ISO shortcode for Ukraine `UA`? .. UK might get confused with United Kingdom

**Review by joscdk — COMMENTED** [review]
Looks great, thanks for this, just one minor comment

File: src/i18n/index.js (hunk 1 above)
Comment (Mordil) [reply src/i18n/index.js:9]: I believe that's the shortcode for the country, not necessarily the language. The ISO 639-1 table found on Wikipedia shows that the Ukrainian language ID is `uk`.

`uk` in reference to United Kingdom comes into play with the `-` in the language ID: `en-US`, `en-UK` to denote the dialect I believe.

Comment (joscdk) [reply src/i18n/index.js:9]: @Mordil argh didn't see the ISO was part of the README .. I have always been using `ISO-3166` which specifies Ukraine as UA :)

**Review by Mordil — DISMISSED** [review]

**Comment by tanner0101** [comment]
`@invoodoo` from Vapor Discord's Ukrainian channel said they would take a look.

File: src/i18n/uk.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    bio: {
+        first: 'Неблокуюча, подіє-орієнтована архітектура, побудована на основі фреймворка <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO</a> від Apple, забезпечує <a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">високу продуктивність</a>.',
+        second: 'Написаний за допомогою <a target="_blank" href="https://swift.org">Swift</a>, потужній та сучасній мові програмування, яку легко освоїти.',
+        third: 'Виразний, <a target="_blank" href="https://developer.apple.com/videos/play/wwdc2015/408/">протокол-орієнтований дизайн</a> з акцентом на безпеку типів і якість підтримки коду.'
+    },
+    contribute: {
+        title: 'Зроби свій внесок!',
+        options: {
+            code: {
+                description: 'Пропонуй ідеї, логуй баги, або ж створюй пул ріквести в GitHub!'
```
Comment (KateKravchyshyn) [inline src/i18n/uk.js]: 'Пропонуй ідеї, логуй баги або ж створюй пул ріквести в GitHub!'

File: src/i18n/uk.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            },
+            chat: {
+                title: 'Приєднайся до чату',
+                description: 'Допоможи новачкам освоїти Vapor.'
+            },
+            donate: {
+                title: 'Стати спонсором',
+                description: 'Зроби одноразовий внесок, або ж стань постійним спонсором щоб підтримати проект'
```
Comment (KateKravchyshyn) [inline src/i18n/uk.js]: 'Зроби одноразовий внесок або ж стань постійним спонсором, щоб підтримати проект'

File: src/i18n/uk.js (hunk 2 above)
Comment (3a4oT) [reply src/i18n/uk.js]: @KateKravchyshyn  Good catch, thanks!

**Comment by KateKravchyshyn** [comment]
looks good despite minor comments

**Comment by 3a4oT** [comment]
@tanner0101 addressed comments from @KateKravchyshyn and rebased.

**Comment by invoodoo** [comment]
Suggestion to change "логуй баги" to "повідомляй про помилки" and "створюй пул ріквести" to "обеднуй програмний код". Looks like last one was changed to Merge instead of Pull Requests.

**Comment by tanner0101** [comment]
@KateKravchyshyn or @invoodoo, please mark "approved" using GitHub's PR review feature if this looks good now, then we can merge.

**Review by KateKravchyshyn — APPROVED** [review]

**Review by tanner0101 — APPROVED** [review]
Thanks!

**Comment by penny-coin** [comment]
Hey @3a4oT, you just merged a pull request, have a coin! 

You now have 1 coins.

**Comment by tanner0101** [comment]
screen shot 2019-02-27 at 9 31 41 am

**Pull request merged**